tests/src/test/java/io/streamnative/pulsar/handlers/kop/MetricsProviderTest.java

```diff
@@ -166,8 +166,9 @@ public void testMetricsProvider() throws Exception {
         Assert.assertTrue(sb.toString().contains("request=\"ApiVersions\""));
         Assert.assertTrue(sb.toString().contains("request=\"ListOffsets\""));
         Assert.assertTrue(sb.toString().contains("request=\"Fetch\""));
-        Assert.assertTrue(sb.toString().contains("kop_server_REQUEST_LATENCY{success=\"true\",quantile=\"0.99\", "
-                + "request=\"Produce\"}"));
+        // The followed check may fail in CI environment, comment it first
+        //Assert.assertTrue(sb.toString().contains("kop_server_REQUEST_LATENCY{success=\"true\",quantile=\"0.99\", "
+        //        + "request=\"Produce\"}"));
```

Comment:
I commented this line because it's very likely to fail in CI environment. It looks like this check is too strict. Do you have any idea? @hangc0276 

Here is the full metric string: [metrics-string.log](https://github.com/streamnative/kop/files/6947504/metrics-string.log)

Reply:
@BewareMyPower Could you please raise up an issue to track the metrics fix? So that we can unblock this PR.

Reply:
https://github.com/streamnative/kop/issues/643